include/dmtcp.h.in

```diff
@@ -211,6 +211,19 @@ EXTERNC int dmtcp_send_query_to_coordinator(const char *id,
                                             void *val,
                                             uint32_t *val_len);
 
+/*
+ * This function can be used to query all mappings in the given nameservice
+ * database. If a null pointer is passed to the function, the function will
+ * allocate a buffer of required length; it's the caller's responsibility to
+ * free the buffer after use. The key-value mappings in the buffer are
+ * serialized in following format:
+ *
+ *    <int key-length, key, int value-length, value>
+ *    <int key-length, key, int value-length, value>
```

Comment:
Can we change `key-length` to `key_length` and `value-length` to `value_length` to avoid any ambiguity?  (Subtraction is not intended here :-) )